docs/docs/getting-started/studio/01-editor.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 1
+title: Application editor
+---
+
+When an application is created or selected for editing, the following application editor screen will appear.
+
+![aica-studio-application-editor](./assets/aica-studio-application-editor.png)
+
+On the left of the screen is the YAML code representation of the application, which is how applications are stored and
+parsed. On the right are visualization tools, including the unique application graph editor and the 3D scene viewer.
+
+The white chevrons in the middle of the screen can be used to selectively expand the left or right portions of the
+screen.
+
+Application controls are shown above the YAML code.
+
+Exit back to the application manager screen using the X button on the top left. It will prompt to save or discard
```

Comment:
This reminds me, closing unsaved running applications will create a weird overlay of the two modals (something we should look at). Running applicatoins will also not be stopped automatically